Changelog.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+### Changed
+- Update device traits to show PPK devices, but not all serialport capable devices.
```

Comment:
Users don't know about “device traits”, so lets rephrase to something they can understand better. Suggestion:
```suggestion
-   Hide some non-PPK devices in the device list (we are not yet able to limit
    it to strictly only PPK devices).
```